inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImpl.java

```diff
@@ -799,6 +801,48 @@ private KBHandle update(KnowledgeBase kb, UpdateAction aAction)
         return handles;
     }
 
+    @Override
+    public KBStatement getExistingStatement(KnowledgeBase kb, KBHandle aInstance,
+        KBHandle aProperty, String aObject)
+    {
+        if (existThisStatement(kb, aInstance, aProperty, aObject)) {
+            KBStatement kbStatement = new KBStatement();
+            kbStatement.setInstance(aInstance);
+            kbStatement.setProperty(aProperty);
+            kbStatement.setValue(aObject);
+            return kbStatement;
+        }
+
+        return null;
+    }
+
+    public boolean existThisStatement(KnowledgeBase kb, KBHandle aInstance, KBHandle aProperty,
```

Comment:
`statementExists`

I did it also but differently in my qualifier branch. Yours will not work with reification, as the `KBStatement`s are split into several `Statement`s.

Reply:
I have removed it.